.changeset/sour-donkeys-draw.md

```diff
@@ -0,0 +1,5 @@
+---
+"@nomicfoundation/edr": minor
+---
+
+Renamed `json` field in response to `data`
```

Comment:
Apart from this, the minor version change should also signal the introduction of `string | any` as a return type